### PR TITLE
add easyconfig for networkx

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: quick review?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8c1cf71576f1aac011d9 for a full test report.

File: easybuild/easyconfigs/n/networkx/networkx-1.9.1-intel-2015a-Python-2.7.9.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+homepage = 'https://pypi.python.org/pypi/networkx'
+description = """NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics,
+ and functions of complex networks."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
```
Comment (wpoely86) [inline easybuild/easyconfigs/n/networkx/networkx-1.9.1-intel-2015a-Python-2.7.9.eb]: maybe we should start looking into simplifying this for `PythonPackage` easyblock? In principle, only the python version should be specified.

**Comment by wpoely86** [comment]
looks good

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

File: easybuild/easyconfigs/n/networkx/networkx-1.9.1-intel-2015a-Python-2.7.9.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/n/networkx/networkx-1.9.1-intel-2015a-Python-2.7.9.eb]: Can you open a separate issue for that (or a PR ;-)) in the easyblocks repo?

**Pull request merged**